actors/builtin/miner/policy.go

```diff
@@ -6,23 +6,19 @@ import (
 	power "github.com/filecoin-project/specs-actors/actors/builtin/power"
 )
 
-// An approximation to chain state finality.
+// An approximation to chain state finality (should include message propagation time as well).
 const ChainFinalityish = abi.ChainEpoch(500) // PARAM_FINISH
 
-// Lookback from current epoch from which to obtain a PoRep challenge.
-// TODO: HS why is this value unused?
-const PoRepLookback = ChainFinalityish // Should be approximately chain ~finality. PARAM_FINISH
-
-// Minimum and maximum delay (inclusive) between a sector pre-commitment and corresponding proof of commitment.
-const PoRepMinDelay = abi.ChainEpoch(5)  // PARAM_FINISH
-const PoRepMaxDelay = abi.ChainEpoch(10) // PARAM_FINISH
```

Comment:
I assume you added this @jzimmerman ?

I want to remove in favor of the `MaxSealDuration` param, which is more general (PoRep could be non-interactive)...

Specifically while `MaxSealDuration` is larger than `PoRepMaxDelay` (it is time for whole Seal so both for the pre-commit and proof, not just between the two (ie just the proof)), I think the time to PreCommit is comparatively negligible so we can use the single param... Initially we used MaxSealDuration to ensure the seal is tied to the chain using `SealRandEpoch` appropriately even though SRE is unverifiable.

Likewise removed the MinDelay since I couldn't fully see its purpose security wise... Lmk

Reply:
Don't remove MinDelay, needed to prevent potential grinding issue

Reply:
nvm that is the `PreCommitChallengeDelay`